packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -362,18 +375,46 @@ export type PostIsCriticismRequest = {
 
 addGraphQLResolvers({
   Query: {
-    async UserReadHistory(root: void, args: {limit: number|undefined}, context: ResolverContext) {
-      const { currentUser, repos } = context
+    async UserReadHistory(
+      root: void,
+      args: {
+        limit: number,
```

Comment:
I believe this is `number | null` (and the other top-level args are similarly `| null` rather than optional)